### ISSUE TITLE
Accessibility: Fix Postgres query editor fastpass issues

### ISSUE THREAD

**Issue by kaydelaney** [grafana/grafana#42722]

**Comment by codeincarnate** [comment]
Closing as this should be fixed by #43981
